src/LongRunning/OperationsClient.php

```diff
@@ -435,7 +478,11 @@ public function cancelOperation($name, $optionalArgs = [])
      * Sample code:
      * ```
      * try {
-     *     $operationsClient = new OperationsClient();
+     *     $options = [
+     *         'serviceAddress' => 'my-service-address',
+     *         'scopes' => ['my-service-scope'],
+     *     ];
+     *     $operationsClient = new OperationsClient($options);
```

Comment:
LRO doesn't have a service address - isn't there special logic in toolkit for that case?